client/rest-high-level/src/test/java/org/elasticsearch/client/SearchIT.java

```diff
@@ -269,7 +269,7 @@ public void testSearchWithMatrixStats() throws IOException {
         assertEquals(5, matrixStats.getFieldCount("num2"));
         assertEquals(29d, matrixStats.getMean("num2"), 0d);
         assertEquals(330d, matrixStats.getVariance("num2"), 0d);
-        assertEquals(-0.13568039346585542, matrixStats.getSkewness("num2"), 0d);
+        assertEquals(-0.13568039346585542, matrixStats.getSkewness("num2"), 1.0e-16);
```

Comment:
For whatever reason only `getSkewness()` has less precision on `aarch64`:

```
amd64:   -0.13568039346585542
aarch64: -0.1356803934658554
```

It still passes on Intel, because the delta is still valid. It makes the assertion slightly *more* permissive, not less.  I scratched my head over this one as well, curious why only this assertion needed a delta. I asked @jpountz about it. 😄